packages/clarity-js/src/data/metadata.ts

```diff
@@ -8,11 +8,10 @@ import * as metric from "@src/data/metric";
 import { set } from "@src/data/variable";
 
 export let data: Metadata = null;
-export let callback: MetadataCallback = null;
+export let callback: MetadataCallbackOptions[] = [];
```

Comment:
>callback [](http://example.com/codeflow?start=11&length=8)

Shouldn't this be named "callbacks" (plural) as it's a list of registered callbacks.